### DISCUSSION TITLE
How to revert to default

### DISCUSSION THREAD

**Discussion by HB9IIU** [AlexandreRouma/SDRPlusPlus#971] [Q&A]
While getting familiar with SDR++, I "screwed up" in changing some font settings in the DearImGUI Style editor under "debug"
Now I have huge fonts which makes the GUI unusable.
Tried to totally unsinstall and re-isntall, but there must be a file somewehere that keeps records of my mistake.
I am on MacOS.
Is there a file to earse to return to default?
Thanks
Daniel

**Comment by AlexandreRouma** [comment]
normally just restarting the app should be enough.
Otherwise, delete the .app and redownload.
If you still have the legacy .pkg install, get the nightly .app instead https://www.sdrpp.org/nightly

**Comment by michaelgeorge001** [comment]
Remove the '/Users/<your username>/Library/Application Support/sdrpp/config.json' file to reset the SDR++ user window.
Remove the '/Users/<your username>/Library/Application Support/sdrpp' folder to reset everything to the SDR++  defaults - this will REMOVE all user added data.

-Mike-

**Reply by AlexandreRouma** [reply]
No. ImGui doesn't use those files for any of these settings.

**Reply by michaelgeorge001** [reply]
Hi Alexandre,

You are absolutely right, because I misread the question.

If anybody is interested, those are the locations for resetting the "SDR++" configuration files on a Mac.

-Mike-

**Reply by HB9IIU** [reply]
Thank You all. Deleting /Users//Library/Application Support/sdrpp/ worked for me.
